README.md

```diff
@@ -47,3 +47,15 @@ If you need these features combine it with another IoC container such as
 
 - File issues at https://github.com/laminas/laminas-di/issues
 - Documentation is at https://docs.laminas.dev/laminas-di/
+
+## Benchmarks
+
+We provide scripts for benchmarking laminas-di using the
+[PHPBench](https://github.com/phpbench/phpbench) framework; these can be
+found in the `benchmarks/` directory.
+
+To execute the benchmarks you can run the following command:
+
+```bash
+$ vendor/bin/phpbench run --report=aggregate
+```
```

Comment:
@Ocramius Thanks for your time and review. I'll get back to resolving the things you mentioned later today!

Reply:
@sjokkateer has done everything right so far, because he has taken the text from other packages.
But it is correct to add the text to centralized location like https://github.com/laminas/.github/blob/-/CONTRIBUTING.md